hls4ml/writer/vivado_accelerator_writer.py

```diff
@@ -139,10 +163,12 @@ def write_axi_wrapper(self, model):
                     newline += indent + '}\n'
                 elif io_type == 'io_stream':
                     newline = ''
+                    newline += 'LOAD_INPUT_OUTER_LOOP:\n'
```

Comment:
What's the point of this?

Reply:
Also, can we remove the explicit creation of loops and move that to the template, as suggested above?